index.html

```diff
@@ -47,6 +47,163 @@ <h1>I’m&nbsp;Ramoun Glad&nbsp;to&nbsp;see you!</h1>
+              <time class="category line-height gray" datetime="2015-10-22">2015</time>
+            </li>
+          </ul>
+        </header>
+        <div class="content hero">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="tags flex">
+            <li><a class="line-height" href="#">html</a></li>
+            <li><a class="line-height" href="#">css</a></li>
+            <li><a class="line-height" href="#">javaScript</a></li>
+          </ul>
+        </div>
+        <footer>
+          <button class="bg" type="button">see project</button>
+        </footer>
+      </article>
+      <!-- card/end -->
+
+      <!-- card/begin -->
+      <article class="card flex bg">
+        <header>
+          <img src="./images/snapshoot/snapshoot_2.svg" alt="">
+          <h3 class="black bold cap">multi-post stories</h3>
+          <ul class="info flex">
+            <li><a href="#" class="category ellipse line-height highlight upper">ramoun</a></li>
+            <li><a href="#" class="category ellipse line-height gray cap">back end dev</a></li>
+            <li>
+              <time class="category line-height gray" datetime="2015-10-22">2015</time>
+            </li>
+          </ul>
+        </header>
+        <div class="content hero">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="tags flex">
+            <li><a class="line-height" href="#">html</a></li>
+            <li><a class="line-height" href="#">css</a></li>
+            <li><a class="line-height" href="#">javaScript</a></li>
+          </ul>
+        </div>
+        <footer>
+          <button class="bg" type="button">see project</button>
+        </footer>
+      </article>
+      <!-- card/end -->
+
+      <!-- card/begin -->
+      <article class="card flex bg">
+        <header>
+          <img src="./images/snapshoot/snapshoot_3.svg" alt="">
+          <h3 class="black bold cap">tonic</h3>
+          <ul class="info flex">
+            <li><a href="#" class="category ellipse line-height highlight upper">ramoun</a></li>
+            <li><a href="#" class="category ellipse line-height gray cap">back end dev</a></li>
+            <li>
+              <time class="category line-height gray" datetime="2015-10-22">2015</time>
+            </li>
+          </ul>
+        </header>
+        <div class="content hero">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="tags flex">
+            <li><a class="line-height" href="#">html</a></li>
+            <li><a class="line-height" href="#">css</a></li>
+            <li><a class="line-height" href="#">javaScript</a></li>
+          </ul>
+        </div>
+        <footer>
+          <button class="bg" type="button">see project</button>
+        </footer>
+      </article>
+      <!-- card/end -->
+
+      <!-- card/begin -->
```

Comment:
- [ ] [OPTIONAL] Please let's keep our code less noisy by removing comments 😉

Reply:
Thank you, so much. How can I contact if I need more tips about clean code? :heart: